scripts/main/lychee.js

```diff
@@ -268,6 +284,13 @@ lychee.init = function () {
 			lychee.swipe_tolerance_x = validatedSwipeToleranceX;
 			lychee.swipe_tolerance_y = validatedSwipeToleranceY;
 
+			lychee.nsfw_visible = (data.config.nsfw_visible && data.config.nsfw_visible === "1") || false;
```

Comment:
Not for this PR, but in general there is a *lot* of duplication here :(

Reply:
I know :(